Documentation [16503] (#64)
eProsima/AML-IP: amlip_docs/rst/user_manual/scenarios/scenarios.rst

.. include:: ../../exports/alias.include

.. _user_manual_scenarios:

################
AML-IP Scenarios
################

> The |amlip| framework is divided in different **scenarios** or **use cases** to fulfill all the necessities
> for an :term:`AML` network.

Reviewer: ```suggestion
The |amlip| framework is divided in different **scenarios** or **use cases** that allow it to exploit all the capabilities :term:`AML`.
```

> These scenarios work independently one to each other and make sense separately, but they can be perfectly integrated
> one to each other in order to create a complex network.

Reviewer: ```suggestion
These scenarios work independently of each other and make sense separately, but can be seamlessly integrated with each other to create a more complex network.
```

Each of the scenarios rely on a different set of **Nodes** that perform the different actions required.

> In order to know more about the *Node* concept and their kinds, refer to :ref:`user_manual_nodes` section.

Reviewer: ```suggestion
In order to know more about the *Node* concept and their kinds, please refer to :ref:`user_manual_nodes` section.
```
